adal/tests/Test.ADAL.NET.UIAutomation/CoreTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Test.ADAL.NET.UIAutomation.Infrastructure;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+	public class CoreMobileTests
+	{
+        public static void AcquireTokenTest(ITestController controller)
+		{
+            var user = controller.GetUser(
+                new UserQueryParameters
+                {
+                    IsMamUser = false,
+                    IsMfaUser = false,
+                    IsFederatedUser = false
+                });
+
+            controller.Tap("secondPage");
+            controller.Tap("acquireToken");
+            controller.EnterText("i0116", user.Upn, true);
```

Comment:
>EnterText [](start = 23, length = 9)

is there a faster way to insert text in a field? e.g. via copy/paste? when watching the automation running this seems to take forever.